pmoves/scripts/validate-ollama-inventory.py

```
@@ -0,0 +1,184 @@
+        elif isinstance(node, list):
+            for item in node:
+                walk(item, path_segments)
+
+    walk(profile)
+    return names
+
+
+def fetch_ollama_models(url: str) -> list[dict]:
+    """Hit Ollama's ``/api/tags`` and return the models list."""
+    req = urllib.request.Request(f"{url.rstrip('/')}/api/tags",
+                                  headers={"Accept": "application/json"})
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)